rails.gemspec

```diff
@@ -27,6 +27,6 @@ Gem::Specification.new do |s|
   s.add_dependency 'activejob',     version
   s.add_dependency 'railties',      version
 
-  s.add_dependency 'bundler',         '>= 1.3.0', '< 2.0'
+  s.add_dependency 'bundler',         '>= 1.9.0', '< 2.0'
```

Comment:
Try to change the Gemfile without updating the bundler, you can understand the reason. :-)

```
`mswin64` is not a valid platform. The available options are: [:ruby, :ruby_18, :ruby_19, :ruby_20, :ruby_21, :mri, :mri_18, :mri_19, :mri_20, :mri_21, :rbx, :jruby, :jruby_18, :jruby_19, :mswin, :mingw, :mingw_18, :mingw_19, :mingw_20, :mingw_21, :x64_mingw, :x64_mingw_20, :x64_mingw_21]
```